csharp/ql/test/library-tests/constructors/PrintAst.expected

```diff
@@ -25,6 +25,7 @@ constructors.cs:
 #   23|           -1: [TypeMention] object
 #   23|         1: [Parameter] s
 #   23|           -1: [TypeMention] string
+#   23|       4: [BlockStmt] {...}
```

Comment:
For generated expressions, we're using `expr_compiler_generated`. Should we mark `SyntheticEmptyBlock`s as compiler generated too?

Reply:
We can add it to the `compiler_generated` table?

Reply:
Hmm, that will require database changes.

Reply:
We could create `stmt_compiler_generated`. Should we do that?

Reply:
I think it would be fine to do this in a separate PR, so that you could merge this one. 

Is there any benefit of having `expr_compiler_generated`, `compiler_generated`, and `stmt_compiler_generated`? It looks like any solution would need a DB schema change. I think my preference would be merging all of this into `compiler_generated`.

Reply:
That would be a nice simplification in any case. I will add that to my list of follow up items!